core/src/test/resources/sql/misc.iq

```diff
@@ -1689,7 +1689,7 @@ EnumerableCalc(expr#0=[{inputs}], expr#1=[123:BIGINT], EXPR$0=[$t1])
 !plan
 
 # Cast an integer literal to a decimal; note: the plan does not contain CAST
-values cast('123.45' as decimal(4, 2));
+values cast('123.45' as decimal(5, 2));
 +--------+
```

Comment:
Why change the query ? It works in MySql and PostgresSQL.

Reply:
It overflows.
```
chunwei.lcw@chunwei:~$ psql
psql (12.3)
Type "help" for help.

chunwei.lcw=# select cast('111.23' as decimal(4, 2));
ERROR:  numeric field overflow
DETAIL:  A field with precision 4, scale 2 must round to an absolute value less than 10^2.
```

Reply:
Hmm, it seems only MySql succeed, with a result of `99.99` for query `select cast('123.45' as decimal(4, 2))`

Reply:
We may need to have a conclusion for this behavior change, it seems breaking.

Reply:
Yep.